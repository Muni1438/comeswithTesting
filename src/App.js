import './App.scss';
import BasicTesting from './components/BasicTesting';
import ToDoApp from './components/ToDoApp';

function App() {
  return (
    <div className="App">
    { /*<ToDoApp/>*/}
    <BasicTesting/>
    </div>
  );
}

export default App;
