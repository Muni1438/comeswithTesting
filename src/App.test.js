  import { render, screen } from '@testing-library/react';
import App from './App';
import ReactDom from "react-dom"

test('renders learn react link', () => {
   const root = document.createElement("div")
   ReactDom.render(<App/>,root)
});
