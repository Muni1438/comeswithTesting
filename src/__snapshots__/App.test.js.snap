// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Take a Snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div>
      <form>
        <h1
          aria-label="h1Content"
          class="tagh1"
        >
          Hello there I'm back, and i'm here testing knowlwdge
        </h1>
        <input
          data-testid="inputTag"
          placeholder="Enter Text"
          type="text"
          value=""
        />
        <input
          data-testid="outputTag"
          placeholder="OutPutTag"
          type="text"
          value=""
        />
        <button>
          Increase
        </button>
        <button>
          Decrease
        </button>
        <button>
          Reset
        </button>
        <br />
        <h1
          aria-label="countMe"
        >
           0
        </h1>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
